Gem file

domducsutcliffe opened the pull request:
Database yml

tberey reviewed (approved):
Approving ahead of time, as the code does look good, despite initial config issues.

domducsutcliffe commented:
In this PR I did more than one thing. 
1) I added postgres to the database .yml
2) I added the pg gem to to the gemfile 
3) I added the original migration files to the db folder.

Herein I will limit my changes to one file per PR.
